### ISSUE TITLE
Vacuolated head

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9840]
**Preferred term label:**
Vacuolated head

**Synonyms**

**Definition (free text, please give PubMed ID)**
For illustration see: WHO laboratory manual for the examination and processing of human semen (6th Edition)

**Parent term (use hpo.jax.org/app)**
HP:0012865

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399808

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
@MagoWyr 
according to several sources, the presence of vacuoles in the sperm head is normal
https://www.hindawi.com/journals/bmri/2014/927841/
There are some articles that suggest that an increased size or count of vacuoles is abnormal, e.g., 
https://pubmed.ncbi.nlm.nih.gov/22559046/

thoughts?

**Comment by MagoWyr** [comment]
Yes, that is correct! Like all abnormalities, it should only be noted if the spermatozoa a uniformly altered close to 100%. If there are some sperm with vacuoles, this is a normal finding. But if <4% of sperm look normal and nearly all sperm show vacuoles, this is a relevant finding.
________________________________

**Comment by pnrobinson** [comment]
@MagoWyr 

I am adding the term as a child of Abnormal sperm head morphology with the following definition. Please reopen this issue if anything needs to be revised further.


Term label: Enlarged sperm head vacuole
Def: Increased size of sperm vacuoles, which are surface concavities typically located at the the tip or middle area of sperm heads. According to the WHO Laboratory Manual for the Examination and Processing of Human Semen, sperm
head vacuoles are considered abnormal when exceeding 20% of the head’s cross-sectional area. Vacuoles are found in almost all samples with normal sperm morphology; this term applies if the proportion of sperm with abnormally large vacuoles is above the upper limit of normal.
PMID:22559046
PMID:24818161
